src/electron.ts

```diff
@@ -115,35 +115,21 @@ app.on('activate', () => {
 });
 
 function registerShortcuts(win: Electron.BrowserWindow): void {
-  globalShortcut.register('CommandOrControl+T', () => win.webContents.send('newTab', true));
-  globalShortcut.register('CommandOrControl+N', () => createWindow());
-  globalShortcut.register('CommandOrControl+Shift+Left', () => win.webContents.send('tabLeft', true));
-  globalShortcut.register('CommandOrControl+Shift+Right', () => win.webContents.send('tabRight', true));
-  globalShortcut.register('CommandOrControl+1', () => win.webContents.send('switchTab', 0));
-  globalShortcut.register('CommandOrControl+2', () => win.webContents.send('switchTab', 1));
-  globalShortcut.register('CommandOrControl+3', () => win.webContents.send('switchTab', 2));
-  globalShortcut.register('CommandOrControl+4', () => win.webContents.send('switchTab', 3));
-  globalShortcut.register('CommandOrControl+5', () => win.webContents.send('switchTab', 4));
-  globalShortcut.register('CommandOrControl+6', () => win.webContents.send('switchTab', 5));
-  globalShortcut.register('CommandOrControl+7', () => win.webContents.send('switchTab', 6));
-  globalShortcut.register('CommandOrControl+8', () => win.webContents.send('switchTab', 7));
-  globalShortcut.register('CommandOrControl+9', () => win.webContents.send('switchTab', 8));
-  globalShortcut.register('CommandOrControl+0', () => win.webContents.send('switchTab', 9));
+  shortcuts.forEach( shortcut => {
+    if (shortcut.sctype === 'webcontents'){
+      globalShortcut.register(shortcut.keypress, () => win.webContents.send(shortcut.sckey, shortcut.scvalue));
+    } else if (shortcut.sctype === 'function'){
+      if (shortcut.sckey === 'createWindow'){
```

Comment:
How about improving this a bit further...

Creating an `obj` of functions or even better a class, i.e. 
````
// keypressFunctions
{
  'send': function(key, value){ win.webContents.send(key, value)}
  'toggleDevTools': function(){ win.webContents.toggleDevTools(); },
  'createWindow': function(){ createWindow(); }
}
```
Then you could also change this `if` statement to `switch` and write something like:
```
shortcuts.forEach( shortcut => {
  globalShortcut.register(shortcut.keypress, () => keypressFunctions[shortcut.scfunction]);
});
```
... maybe there is no need for `sctype`.

What do you think?